min/config.php

```diff
@@ -65,7 +65,7 @@
  * second line. The third line might work on some Apache servers.
  */
 $min_documentRoot = '';
-//$min_documentRoot = substr(__FILE__, 0, -15);
+$min_documentRoot = substr(__FILE__, 0, -15);
 //$min_documentRoot = $_SERVER['SUBDOMAIN_DOCUMENT_ROOT'];
```

Comment:
Getting bad requests for:
/fonts/fontawesome-webfont.woff?v=4.5.0 in my website 
Please suggest me what to do